monolithe/generators/lang/java/templates/sdkinfo.java.tpl

```diff
@@ -0,0 +1,16 @@
+{{ header }}
+
+package {{ name }}.{{ version_string }};
+
+public interface SdkInfo {
+
+    double API_VERSION      = {{ version }};
+    String API_PREFIX       = "{{ api_prefix }}";
+    String PRODUCT_ACCRONYM = "{{ product_accronym }}";    
+    String PRODUCT_NAME     = "{{ product_name }}";    
+    String CLASS_PREFIX     = "{{ class_prefix }}";    
+    String NAME             = "{{ name }}";
+    String ROOT_API         = "{{root_api}}";
+    Class<?> ROOT_OBJECT_CLASS  = {{ class_prefix }}{{ root_api|capitalize }}.class;
```

Comment:
It's a mistake then. Our root api happens to be `me`, so capitalize will work. but it's RootObject, that won't work. I guess that's fine for now, but remember Monolithe should not care about VSD at all :)